requirements.in

```diff
@@ -1,17 +1,16 @@
-django==3.2.25
-requests==2.31.0
-requests_oauthlib
+django==4.2.20
+requests==2.32.3
+requests_oauthlib==2.0.0
 django-dotenv==1.4.2
-sentry-sdk==1.39.1
```

Comment:
Thanks for removing this package as this project isn't actively using Sentry anymore.